manifests/params.pp

```diff
@@ -404,6 +404,12 @@
     $safe_trap_service_hasrestart = $trap_service_hasrestart
   }
 
+  if $snmp_openmanage_enable {
+    $snmpv2_enable =  $::snmp_snmpv2_enable
```

Comment:
where is ```$snmp_snmpv2_enable``` coming from? Shouldn't we just do:
```
$snmpv2_enable =  false
```
?

Reply:
I think you are right; This parameter come from init.pp and default value is in params.pp. I'll fix this.